src/pages/ConfigRoom.js

```diff
@@ -122,29 +122,26 @@ export default function ConfigRoom() {
             placeholder={defVotesToSkip}
           ></Input>
         </Votes>
-        <RoomButton onClick={() => setIsReadyToCreateRoom(true)}>
-          ¡Crear sala!
-        </RoomButton>
-        <BackButton onClick={() => history.push("/")}>Return</BackButton>
+        <RoomButton>¡Crear sala!</RoomButton>
```

Comment:
Aqui se esta perdiendo el tracking de cuando el usuario ya da la orden de crear la sala.
```suggestion
        <RoomButton onClick()={() => setIsReadyToCreateRoom(true)}}>¡Crear sala!</RoomButton>
```